cfme/infrastructure/virtual_machines.py

```diff
@@ -132,24 +132,23 @@ def _nav_to_snapshot_mgmt(self):
                 sel.click(InfoBlock.element("Properties", "Snapshots"))
 
         def does_snapshot_exist(self):
+            name = self.name or self.description
```

Comment:
Are you sure about the precedence order in here? I know that in ManageIQ usually description is the one that takes precedence when specified.